versioned_docs/version-8.8/self-managed/components/components-upgrade/870-to-880.md

```diff
@@ -135,6 +140,45 @@ When you enable the Identity migration, the Camunda Helm chart automatically con
   </TabItem>
 </Tabs>
 
+#### Entity IDs configuration
+
+The Identity Migration Application requires consistent entity IDs between Management Identity and Orchestration Cluster Identity to ensure a smooth migration. To ensure this, you can configure the following properties matching the values used in your Management Identity setup.
+
+<Tabs>
+  <TabItem value="env" label="Environment variables" default>
+
+| Environment variable                                        | Description                                                                                                                     | Default value   |
+| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- | --------------- |
+| `CAMUNDA_MIGRATION_IDENTITY_ENTITIES_DEFAULT_LOWERCASE`     | If true, the IDs of ROLES, GROUPS, MAPPING RULES and USERS will be lowercased. Specific entities configuration have precedence. | `true`          |
+| `CAMUNDA_MIGRATION_IDENTITY_ENTITIES_DEFAULT_PATTERN`       | Configure the pattern that the IDs have to respect. Specific entities configuration have precedence.                            | `[^a-z0-9_@.-]` |
```

Comment:
Let's explain here that this pattern must not be more permissive than the Orchestration Cluster's ID id validation pattern documented here: https://docs.camunda.io/docs/self-managed/components/orchestration-cluster/core-settings/configuration/properties/#camundasecurity

Otherwise the migration app will fail when it attempts to create such entities. Users should be aware that they may have to change both properties in sync.

Same for the other pattern properties here.